GameServerLib/Content/CharData.cs

```diff
@@ -77,7 +86,7 @@ public class CharData : ICharData
             new[] {6, 11, 16, 99, 99, 99}
         };
 
-        public string[] AttackNames { get; private set; } = new string[18];
+        public List<IBasicAttackInfo> BasicAttacks { get; private set; } = new List<IBasicAttackInfo>(new IBasicAttackInfo[18]);
```

Comment:
Why not just `new List<IBasicAttackInfo>(18)`?

Reply:
![image](https://user-images.githubusercontent.com/72713387/180573061-723f80d7-c231-4f42-ac63-51b288f0fa7a.png)